allofplos/plos_corpus.py

```diff
@@ -385,7 +385,7 @@ def download_amended_articles(directory=None, tempdir=newarticledir, amended_art
     return amended_updated_article_list
 
 
-def get_uncorrected_proofs_list(directory=None):
+def get_uncorrected_proofs_list(directory=None, text_list=uncorrected_proofs_text_list):
```

Comment:
I think that you should make the text_list parameter more meaningful… it’s a filepath, right?